aten/src/ATen/core/function_schema.cpp

```diff
@@ -41,9 +41,15 @@ FunctionSchema FunctionSchema::cloneWithRealTypes(bool with_symint) const {
     }
   };
   std::vector<Argument> new_arguments, new_returns;
-  std::transform(arguments().begin(), arguments().end(), std::back_inserter(new_arguments), cloneWithRealTypes);
+  new_arguments.reserve(arguments().size());
+  for (const auto& arg: arguments()) {
+    new_arguments.push_back(cloneWithRealTypes(arg));
```

Comment:
Is there are reason to prefer `push_back` to `emplace_back`
```suggestion
    new_arguments.emplace_back(cloneWithRealTypes(arg));
```

Reply:
push_back has (since C++11) an overload that takes T&&. emplace_back does more complicated shenanigans with forwarding its arguments to the constructor of T. here we have a T&&, and so push_back matches our needs.

Reply:
see also https://quuxplusone.github.io/blog/2021/03/03/push-back-emplace-back/